CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## [0.1.5]
+
+### Fixed
+
+* Fixed AlignmentChart error in which `colorscale` being undefined
+  caused rendering to fail.
+* Fixed AlignmentChart PropType for `eventDatum` to `string` (which is
+  what gets returned).
+* Fixed Circos error in which `tracks` variable being undefined caused
+  rendering to fail.
+* Fixed Clustergram error in which `display_ratio` (a float or a list)
+  was used as a list instead of `self._display_ratio`, which always
+  has the type `list`.
+* Fixed Molecule2dViewer error in which `modelData` being undefined
+  causesd rendering to fail.
+* Fixed Molecule2dViewer PropType for `source` and `target` atoms
+  within the `links` property to also include `PropTypes.shape`.
+* Fixed Molecule3dViewer error in which there was no re-rendering upon
+  a change in the `selectionType` prop.
+* Fixed SequenceViewer error in which the component initially rendered
+  with coverage that had no defined `onClick` behavior.
+
+### Added
+
+* Added re-rendering for Molecule2dViewer and Molecule3dViewer upon
+  the `selectedAtomIds` prop being updated. (This allows for the user
+  to write to the `selectedAtomIds` prop through a Dash component.)
+* Added re-rendering for SequenceViewer upon the `coverageClicked`
+  prop being updated.
+
+## [0.1.4] - 2019-08-02
+
+### Fixed
+* Fixed incompatibility with Dash 1.0.0.
```

Comment:
Hmm, I would put this under "Changed" and call it "Migrated to React 16 and Dash 1" :)

Reply:
I think it makes more sense to put it under "fixed", since this library can't really be used without Dash and it *is* an error for it to be out-of-date.

Reply:
Would make this `Changed`. Moving from `dash==0.40` to `dash>=1.0.0` is not a bug fix but updating one of the library's core dependency. As such,React 16 can be mentioned but is just a side-effect of updating Dash compatibility and I would personally skip it.